owncloudData/src/main/java/com/owncloud/android/data/ProviderMeta.java

```diff
@@ -31,14 +31,15 @@
 
     public static final String DB_NAME = "filelist";
     public static final String NEW_DB_NAME = "owncloud_database";
-    public static final int DB_VERSION = 31;
+    public static final int DB_VERSION = 32;
 
     private ProviderMeta() {
     }
 
     static public class ProviderTableMeta implements BaseColumns {
         public static final String OCSHARES_TABLE_NAME = "ocshares";
         public static final String CAPABILITIES_TABLE_NAME = "capabilities";
+        public static final String OCFILES_TABLE_NAME = "files";
```

Comment:
These constants names/values are a bit messy, if files table doesn't have `oc` before, you could use `FILES_TABLE_NAME`... It would be great to have everything similar `ocshares`, `occapabilities` and `ocfiles` or without the oc all of them, but I guess that will need a migration

Reply:
You guess right, i can rename it to FILES_TABLE_NAME and keep it the same format as capabilities